### DISCUSSION TITLE
Customize/Add custom class on Wrapper

### DISCUSSION THREAD

**Discussion by aromiko** [TarekRaafat/autoComplete.js#338] [Q&A]
Hi! Just want to ask if I could add a custom class on the wrapper or customize the name of it. We follow BEM style of css and targeting `autoComplete_wrapper` to add a custom property will be against our standard because of its naming. Thanks!

**Comment by folknor** [comment]
It inherits the name from `config.name`; https://tarekraafat.github.io/autoComplete.js/#/configuration?id=name-optional

So if you name your object "foo", the wrappers class will be "foo_wrapper".

Other than that, the answer to your question is "no". But I suspect you didn't want a literal answer to the question :-P

Of course the code can be changed to accomodate you, but currently it does not.
